Makefile

```diff
@@ -265,28 +265,34 @@ install-tools:
 generate:
 	go generate ./...
 
-.PHONY: artifacts-darwin
-artifacts-darwin:
-	mkdir -p _artifacts
-	GOOS=darwin GOARCH=amd64 make clean binaries
-	$(TAR) -C _output/ -czvf _artifacts/lima-$(VERSION_TRIMMED)-Darwin-x86_64.tar.gz ./
-	GOOS=darwin GOARCH=arm64 make clean binaries
-	$(TAR) -C _output -czvf _artifacts/lima-$(VERSION_TRIMMED)-Darwin-arm64.tar.gz ./
+.PHONY: artifacts-darwin artifact-darwin-aarch64 artifact-darwin-arm64 artifact-darwin-x86_64
+artifacts-darwin: artifact-darwin-x86_64 artifact-darwin-arm64
+artifact-darwin-arm64: ENVS=GOOS=darwin GOARCH=arm64
+artifact-darwin-arm64: _artifacts/lima-$(VERSION_TRIMMED)-Darwin-arm64.tar.gz
+artifact-darwin-aarch64: artifact-darwin-arm64
```

Comment:
Since the artifact file names use the values returned by `uname -m` on each platform, it would be better to use the values returned by `uname -m` for the Makefile targets that are likely to run on each platform as well.
`arm64` on Darwin, `aarch64` on Linux.